.gitignore

```diff
@@ -10,7 +10,11 @@ ignored
 !tests/files/f3d9b387e311d5ab59a8c08eb3545dbb.json
 !*.schema.json
 !args.json
-tests/files/poppunk_output
+tests/files/poppunk_output/*
```

Comment:
It might be better to put your files that are under version control at a totally different path. )You can also use `git add -f` to bypass the `.gitignore`). but this way leads to mix of generated and "real" files which is rarely what is wanted

Reply:
I tried to separate this, having (nearly) all generated files in a `results` folder (see this [commit](https://github.com/bacpop/beebop_py/pull/6/commits/a294ce4e694ca4aa63b5fecf53b29e8b3d99336f)).
Main change in the test files is that I'm using 2 different PoppunkFilestores for file input and output. 
The only two test where this was not straightforward are [`test_assign_clusters`](https://github.com/bacpop/beebop_py/blob/a294ce4e694ca4aa63b5fecf53b29e8b3d99336f/tests/test_unit.py#L54) and [`test_microreact_internal`](https://github.com/bacpop/beebop_py/blob/a294ce4e694ca4aa63b5fecf53b29e8b3d99336f/tests/test_unit.py#L73), where input and output cannot be separated easily. 
For `test_assign_clusters` I solved this by copying the files to the output location first to work at this location.
Is this approach sensible and should I do the same for `test_microreact_internal`?

Reply:
Should I do this then for all tests - not just the ones where separation is tricky, to avoid having the two filestores?